stretch-auth/src/App/App.js

```diff
@@ -77,6 +79,10 @@ class App extends Component {
       .catch(error => console.log(error));
   }
 
+  savePokemonToUser(pokemon, slot) {
+    localStorage.setItem(`${this.state.currentUser.displayName} ${slot}`, JSON.stringify(pokemon))
```

Comment:
Good idea saving this to Local Storage.

Reply:
Yea good call only ever having to download once